packages/driver/src/cypress/error_messages.ts

```diff
@@ -325,6 +325,12 @@ export default {
         docsUrl: 'https://on.cypress.io/session',
       }
     },
+    invalid_domain (obj) {
+      return {
+        message: `${cmd('{{cmd}}')} must be passed a valid domain name. You passed: \`{{domain}}\``,
+        docsUrl: `https://on.cypress.io/${_.toLower(obj.cmd)}`,
```

Comment:
only a thought -- I know we had talked about documenting domain / super domain / etc at some point with the origin work. Might be worth pointing to that doc some day to give better reference to whats an appropriate domain.

Reply:
Yep 😄 https://github.com/cypress-io/cypress-documentation/pull/4824#discussion_r1012160257

Reply:
OHH i was actually thinking more of a "what is domian vs superdomain vs url schema type breakdown" 😅

Reply:
Exactly. Once Bill writes that doc we can point to it here and in the docs for this option.